fastly/kafka_test.go

```diff
@@ -38,7 +38,7 @@ func TestClient_Kafkas(t *testing.T) {
 			TLSClientCert:    clientCert,
 			TLSClientKey:     clientKey,
 			ParseLogKeyvals:  true,
-			RequestMaxBytes:  1024,
+			RequestMaxBytes:  2048,
```

Comment:
As this is changing functionality, this feels like it should be in a separate PR and not in a test fixtures update PR. As will make documenting in the change-log easier.

Reply:
Changing the value from `1024` to `2048` was a change that _had_ to happen in order to regenerate the fixture 😬 it seems the backend endpoint changed its behaviour.